plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptions/GroupedProductSelectorItemCTA.php

```diff
@@ -31,12 +31,16 @@ class GroupedProductSelectorItemCTA extends AbstractBlock {
 	 * @return string The HTML markup for the quantity selector.
 	 */
 	private function get_quantity_selector_markup( $product ) {
-		ob_start();
-
-		woocommerce_quantity_input( AddToCartWithOptionsUtils::get_quantity_input_args( $product ) );
+		$args             = AddToCartWithOptionsUtils::get_quantity_input_args( $product );
+		$args['input_id'] = AddToCartWithOptionsUtils::get_quantity_input_id( $product );
```

Comment:
Why are we using different types of ID? The checkbox below use `quantity-<ID>` format. Can we use the same format here?

Reply:
this was the case before, but I agree they should follow the same format. Will that break any kind of campatibility if we change it?

Reply:
> Can we use `quantity-<ID>` for both place?

Done in 85e45cb9bcca8f6d1693b266e53cbaf5960e6e14. I think it should be safe to have the same ID for the two types of inputs.